ppr-api/tests/unit/api/test_search_results.py

```diff
@@ -0,0 +1,219 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to verify the searches endpoint.
+
+Test-Suite to ensure that the /searches endpoint is working as expected.
+"""
+import copy
+from http import HTTPStatus
+
+# prep sample post search data
+from registry_schemas.example_data.ppr import SEARCH_SUMMARY
+
+from ppr_api.services.authz import STAFF_ROLE, COLIN_ROLE, PPR_ROLE
+from tests.unit.services.utils import create_header_account, create_header
+
+
+SAMPLE_JSON_SUMMARY = copy.deepcopy(SEARCH_SUMMARY)
+
+
+def test_search_detail_valid_200(session, client, jwt):
```

Comment:
you could collapse these with pytest parameterized tests

Reply:
Noted. I am refactoring as changes are made to unit test files. These tests already existed.